### PR TITLE
include jupyter notebooks in docs

### PR DESCRIPTION
**Pull request by ltalirz**
* move jupyter notebooks into `notebooks/` subfolder (were difficult to find otherwise)
 * Include Jupyter notebooks that illustrate the use of the widgets in the docs

### CODE REVIEW

**Comment by ltalirz** [comment]
Example:
image


Note that the notebooks currently don't contain outputs.
We may want to change that in the future

**Comment by ltalirz** [comment]
If you want to test it locally, just `pip install -e --use-deprecated=legacy-resolver .[docs]`  and type `make`  in the docs dir

File: docs/source/widget-list/index.rst [hunk 1]
```diff
@@ -2,15 +2,20 @@
 List of widgets
 ***************
 
-Here we collect the information about all the widgets available in AWB and provide a simple example of how to use them.
+Here we collect a set of Jupyter notebooks that illustrate how to use all available widgets.
```
Comment (csadorf) [inline docs/source/widget-list/index.rst:5]:
```suggestion
This is a collection of Jupyter notebooks that illustrate how to use the widgets available from this library.
```

File: setup.cfg [hunk 2]
```diff
@@ -45,6 +45,7 @@ docs =
     sphinx-rtd-theme
     sphinxcontrib-contentui
     sphinxcontrib-details-directive
+    nbsphinx
```
Comment (csadorf) [inline setup.cfg:48]: When I tried to build the docs I also needed to install pandoc, I assume that's related to this? Anyway to document this?

File: notebooks/eln_configure.ipynb [hunk 3]
```diff
@@ -1,5 +1,15 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "understanding-authority",
+   "metadata": {},
+   "source": [
+    "# Configure connection to ELN\n",
+    "\n",
+    "Authenticate the AiiDAlab with an electronic laboratory notebook"
```
Comment (csadorf) [inline notebooks/eln_configure.ipynb:10]:
```suggestion
    "Authenticate AiiDAlab with an Electronic Laboratory Notebook (ELN)"
```

**Review by csadorf — APPROVED** [review]
It would indeed be much better if we also showed the output of those notebooks, but ok to add this in a separate PR.

I have a few suggestions for slightly different wording, but overall good.

File: setup.cfg (hunk 2 above)
Comment (ltalirz) [reply setup.cfg:48]: I see, thanks for pointing this out. I'll document it in the "development installation" 

I'll check, and I'll also check with Chris whether there are now tools that could avoid the pandoc dependency.

Comment (ltalirz) [reply setup.cfg]: @csadorf I now switched to the myst route which should get rid of the pandoc dependency.

**Comment by ltalirz** [comment]
> It would indeed be much better if we also showed the output of those notebooks, but ok to add this in a separate PR.

Let's do this in a separate PR.

Both nbsphinx and myst-nb have machinery for running the notebooks automatically, so we can look into enabling it (although it does not work out of the box)

**Review by csadorf — APPROVED** [review]

**Pull request merged**